### PR TITLE
update meeting schedule

### PR DESCRIPTION
**Pull request by Trott**
* Change meeting cadence from weekly to biweekly. (We've been canceling a lot of meetings lately due to light agendas. This is good. It means we're resolving issues in the issue tracker. A+.)

* Remove meetings that have already occurred.

* Add meetings for July 19 and August 2.

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/ctc

**Review by mcollina — APPROVED** [review]
LGTM

**Review by bnoordhuis — APPROVED** [review]

**Review by ChALkeR — APPROVED** [review]
LGTM, but imo this requires a majority consent before landing.

File: README.md [hunk 1]
```diff
@@ -32,11 +32,7 @@ Meta technical matters to be discussed here include, but are not limited to:
...
 | ---- | -------------- | -------- | --------- | ------ | ------- | -------- | ----- | ------ |
-|Wed, May 31, 2017 at 16:00:00 | Wed 9:00 am * | Wed 12:00 noon * | Wed 6:00 pm * | Wed 7:00 pm | Wed 9:30 pm | Thu midnight | Thu 1:00 am | Thu 2:00 am|
-|Wed, June 7, 2017 at 20:00:00 | Wed 1:00 pm * | Wed 4:00 pm * | Wed 10:00 pm * | Wed 11:00 pm | Thu 1:30 am | Thu 4:00 am | Thu 5:00 am | Thu 6:00 am|
-|Wed, June 14, 2017 at 11:00:00 | Wed 4:00 am * | Wed 7:00 am * | Wed 1:00 pm * | Wed 2:00 pm | Wed 4:30 pm | Wed 7:00 pm | Wed 8:00 pm | Wed 9:00 pm|
-|Wed, June 21, 2017 at 16:00:00 | Wed 9:00 am * | Wed 12:00 noon * | Wed 6:00 pm * | Wed 7:00 pm | Wed 9:30 pm | Thu midnight | Thu 1:00 am |Thu 2:00 am|
-|Wed, June 28, 2017 at 20:00:00 | Wed 1:00 pm * | Wed 4:00 pm * | Wed 10:00 pm * | Wed 11:00 pm | Thu 1:30 am | Thu 4:00 am | Thu 5:00 am | Thu 6:00 am|
-|Wed, July 5, 2017 at 11:00:00 | Wed 4:00 am * | Wed 7:00 am * | Wed 1:00 pm * | Wed 2:00 pm | Wed 4:30 pm | Wed 7:00 pm | Wed 8:00 pm | Wed 9:00 pm|
-
+|Wed July 5, 2017 at 11:00:00 | Wed 4:00 am * | Wed 7:00 am * | Wed 1:00 pm * | Wed 2:00 pm | Wed 4:30 pm | Wed 7:00 pm | Wed 8:00 pm | Wed 9:00 pm|
```
Comment (ChALkeR) [inline README.md:35]: The formatting changed here (and on other lines) — comma got removed: `Wed, July 5` → `Wed July 5`.
I have no opinion on this (this is not even a nit), just noting to confirm if this is intentional or not.

**Review by cjihrig — APPROVED** [review]

**Review by targos — APPROVED** [review]

**Review by joyeecheung — APPROVED** [review]

**Review by evanlucas — APPROVED** [review]

**Pull request merged**